pyproject.toml

```diff
@@ -43,3 +46,6 @@ ignore = [
 line-length = 90
 [tool.ruff.pep8-naming]
 classmethod-decorators = ["classmethod", "pydantic.validator"]
+
+[tool.pytest.ini_options]
+pythonpath = ["."]
```

Comment:
Pretty sure you could narrow this to "./title_belt_nhl", but probably doesn't make a big difference.